Microsoft.Azure.CosmosRepository/src/ChangeFeed/ChangeFeedOptions.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Azure.CosmosRepository.ChangeFeed
+{
+    /// <summary>
+    /// The options for monitoring the change feed.
+    /// </summary>
+    public class ChangeFeedOptions
+    {
+        internal Type ItemType { get; }
+
+        internal ChangeFeedOptions(Type itemType)
+        {
+            ItemType = itemType;
+        }
+
+        /// <summary>
+        /// The instance name of the processor.
+        /// </summary>
+        public string InstanceName { get; set; } = "default";
```

Comment:
This isn't configurable via the options pattern, this can only be configured as shown below

```
_repositoryOptions.ContainerBuilder.Configure<TestItem>(builder =>
{
    builder.WithContainer("a");
    builder.WithChangeFeedMonitoring(options => options.InstanceName = "abc");
});
```

it's on a per-item basis